### ISSUE TITLE
still active ?

### ISSUE THREAD

**Issue by pr0vieh** [freealbion/freealbion#1]
Hi,

i have search for an texture remake of Albion and i found this...
is the project currently still active ?
and how far away from a playable overhaul of albion ?
i love what i can see in pr_1 :heart_eyes:

greets Pr0vieH

**Comment by a2flo** [comment]
No, I'm not actively working on this any more (or at least not right now). I realized a while ago that I would have to rewrite/overhaul the complete 2D rendering system to make it render correctly, especially with upscaled textures and other "modern" niceties like linear filtering and arbitrary scaling, or correct depth layering (then again, the original has its issues with this as well). And I'd need to redo the 3D rendering part as well now, because the engine has become rather dated ...
All of this is obviously a lot of work, and writing OpenGL/GLSL code _for fun_ is just too horrific at this point. I _might_ come back to this if I do write a new Vulkan/Metal-based engine (haven't decided on this yet, will also depend on how much time I have) and if I do consider this project a nice challenge to use it for.

Concerning non-rendering stuff and general "playability": there is a lot of work still to be done. It's essentially just a map viewer right now, with some minor event handling. There are some parts that I've implemented since PR1 (like audio/sound loading and playing, or the basics of a script interpreter), but this isn't actually wired in yet. And huge parts like game state tracking, a dialog system or combat are completely missing. I can't even say if enough of the event system and combat have been reverse engineered to properly reimplement it (probably not). If someone were to release the source code of the original, that would actually go a long way ...

**Comment by piip4** [comment]
You might want to check out Albion port for Windows. It has enhanced graphics (in dungeons) and uses original Albion files. You don't need DosBox to run it.

**Comment by a2flo** [comment]
It's not a remake though. As far as I can tell it's a translated binary with some modifications to it for better rendering.

**Comment by pr0vieh** [comment]
OK thank you for the info
if anyone ever has Bluebyte requested by source code? .... after 21 years ... or maybe they would do a remake they have made of The Settlers II

**Comment by a2flo** [comment]
Technically Ubisoft owns the rights to the source code, and I did write them a mail some time ago, but never got a reply :/ You can certainly try again though.

And I very much doubt they'd do an official remake ... The Settlers is a completely different story (it's actually profitable for Ubisoft/BB). On that note: you can actually buy Albion again on GOG (it's just a DosBox version however).

**Comment by piip4** [comment]
If all you want is a texture remake then that should be doable. I've imported (3D-level) textures into Albion and it seems to work just fine with twice the resolution that Albion originally makes use of.
![3dwalls0 xld entry-13 sprite-0](https://cloud.githubusercontent.com/assets/2265443/13549378/14a4cd78-e304-11e5-91ea-c64ec4ce7914.jpg)

Textures found from Unreal tournament. :dancer: 

![p5060006d2](https://cloud.githubusercontent.com/assets/2265443/13549381/1fe272d0-e304-11e5-8fc2-a0978f63776c.jpg)
